CWE/common/countries/Navajo.txt

```diff
@@ -0,0 +1,153 @@
+color = { 105  145  40 }
+graphical_culture = Generic
+
+party = {
+		name = ""
```

Comment:
This party is missing a name

Reply:
File was included by accident, and is currently an unfinished/unused tag.